app/api/server/api.js

```diff
@@ -15,16 +15,17 @@ import { getDefaultUserFields } from '../../utils/server/functions/getDefaultUse
 
 const logger = new Logger('API', {});
 const rateLimiterDictionary = {};
-const defaultRateLimiterOptions = {
+export const defaultRateLimiterOptions = {
```

Comment:
This was necessary to be exported since else, subclasses can't access it: You cannot make it a member of `APIClass` since it's used in the `super.constructor`